src/id/komikcast/src/eu/kanade/tachiyomi/extension/id/komikcast/KomikCast.kt

```diff
@@ -141,23 +138,8 @@ class KomikCast : MangaThemesia("Komik Cast", "https://komikcast.lol", "id", "/d
     }
 
     override fun pageListParse(document: Document): List<Page> {
-        var doc = document
-        var cssQuery = "div#chapter_body .main-reading-area img"
-        val imageListRegex = Regex("chapterImages = (.*) \\|\\|")
-        val imageListMatchResult = imageListRegex.find(document.toString())
-
-        if (imageListMatchResult != null) {
-            val imageListJson = imageListMatchResult.destructured.toList()[0]
-            val imageList = json.parseToJsonElement(imageListJson).jsonObject
-
-            var imageServer = "cdn"
-            if (!imageList.containsKey(imageServer)) imageServer = imageList.keys.first()
-            val imageElement = imageList[imageServer]!!.jsonArray.joinToString("")
-            doc = Jsoup.parse(imageElement)
-            cssQuery = "img.size-full"
-        }
-
-        return doc.select(cssQuery)
```

Comment:
is this not needed anymore?

Reply:
Yes, the source change it several years ago. Only lasts a few weeks.

The removed code was implemented in mid-June 2022.